xml/src/test/resources/configs/docs/getting-started.xml

```diff
@@ -42,7 +43,7 @@
   <!-- end::gettingStarted[] -->
 
   <!-- tag::expiry[] -->
-  <cache alias="foo">
```

Comment:
The following 2 changes are not consistent with what was done for master. These 2 caches were removed on master.